kura/org.eclipse.kura.web2/src/main/resources/org/eclipse/kura/web/client/messages/Messages.properties

```diff
@@ -144,6 +144,7 @@ fileDownloadSuccess=File was successfully downloaded
 fileDownloadFailure=File download failed
 
 securityIntro=Review and update the available security settings.
+securityCertificateListWarning=Review and update the available certificate list.<br />After you add all the necessary certificates, a device reboot could be required in order for changes to be applied, in case the selected Keystore is not an SSLKeystore or an HttpsKeystore.<br />Update either SSLKeystore or HttpsKeystore could trigger a web console/mqtt data connection restart.
```

Comment:
> available certificate list
available certificates list

> After you add all the necessary certificates, a device reboot could be required in order for changes to be applied, in case the selected Keystore is not an SSLKeystore or an HttpsKeystore.

For some keystore types, a device reboot might be required after a certificate update to make the change effective. For certain services like SSL and Web UI, the corresponding service will automatically restart after a keystore change.